lib/Sema/TreeTransform.h

```diff
@@ -5434,6 +5434,12 @@ QualType TreeTransform<Derived>::TransformTypeOfExprType(TypeLocBuilder &TLB,
 }
 
 template<typename Derived>
+QualType TreeTransform<Derived>::TransformTypeVariableType(TypeLocBuilder &TLB,
+                                                           TypeVariableTypeLoc TL) {
+  return QualType();
```

Comment:
Could yo fill this in, following the pattern for other types?    As far as I can tell, this should just return the underlying type variable.